internal/markup/markup.go

```diff
@@ -160,7 +170,7 @@ func RenderSpecialLink(rawBytes []byte, urlPrefix string, metas map[string]strin
 
 	rawBytes = RenderIssueIndexPattern(rawBytes, urlPrefix, metas)
 	rawBytes = RenderCrossReferenceIssueIndexPattern(rawBytes, urlPrefix, metas)
-	rawBytes = RenderSha1CurrentPattern(rawBytes, urlPrefix)
+	rawBytes = RenderSha1CurrentPattern(rawBytes, urlPrefix, metas)
```

Comment:
In fact, I think the only change we need in this file is this:

```suggestion
	rawBytes = RenderSha1CurrentPattern(rawBytes, metas["repoLink"])
```